<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>GitHub - chrishah/MITObim: MITObim - mitochondrial baiting and iterative mapping</title><style type="text/css"> * {margin:0; padding:0; text-indent:0; }
 .s1 { color: #FFF; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s2 { color: #FFF; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s3 { color: #0366D5; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt; }
 .s5 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s6 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s7 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s8 { color: #575F69; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s9 { color: #575F69; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s10 { color: #444D55; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s12 { color: #596068; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 4pt; }
 .s13 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 11.5pt; }
 .s14 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s15 { color: #FFF; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .p, p { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; margin:0pt; }
 .s16 { color: black; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s17 { color: #23282D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 6pt; }
 .s20 { color: #575F69; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s21 { color: #575F69; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt; }
 .s23 { color: #0366D5; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s24 { color: #575F69; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s25 { color: #6A737C; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s26 { color: #0366D5; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s29 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 18.5pt; }
 .s30 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .a { color: #0366D5; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s31 { color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s32 { color: #23292D; font-family:Consolas, monospace; font-style: normal; font-weight: normal; text-decoration: none; font-size: 8pt; }
 .s33 { color: #0366D5; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 .s34 { color: black; font-style: normal; font-weight: normal; text-decoration: none; }
 .s35 { color: #575F69; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 .s36 { color: #0366D5; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt; }
 li {display: block; }
 #l1 {padding-left: 0pt;counter-reset: c1 1; }
 #l1> li:before {counter-increment: c1; content: counter(c1, decimal)". "; color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 #l1> li:first-child:before {counter-increment: c1 0;  }
 li {display: block; }
 #l2 {padding-left: 0pt;counter-reset: d1 1; }
 #l2> li:before {counter-increment: d1; content: counter(d1, lower-latin)". "; color: #23292D; font-family:"Segoe UI Light", sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt; }
 #l2> li:first-child:before {counter-increment: d1 0;  }
</style></head><body><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s1" style="text-indent: 0pt;text-align: left;"><a name="bookmark0">Why GitHub?</a><a name="bookmark1">&zwnj;</a></p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><a href="https://github.com/enterprise" class="s2">Enterprise</a></p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s1" style="text-indent: 0pt;text-align: left;">Explore</p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><a href="https://github.com/marketplace" class="s2">Marketplace</a></p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s1" style="text-indent: 0pt;text-align: left;">Pricing</p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><a href="https://github.com/join" class="s2" target="_blank">Sign in Sign up</a></p><p style="padding-left: 252pt;text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="789" height="389" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_001.png"/></span></p><p style="padding-top: 11pt;padding-left: 15pt;text-indent: 0pt;text-align: left;"><span style=" color: #0366D5; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt;">chrishah </span><a href="#bookmark1" style=" color: #575F69; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10.5pt;">/ </a><a href="#bookmark1" class="s3">MITObim</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 15pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/watchers" class="s5" target="_blank">Watch 16</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 15pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/stargazers" class="s5" target="_blank">Star 64</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 15pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/network/members" class="s5" target="_blank">Fork 24</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 4pt;padding-left: 8pt;text-indent: 0pt;text-align: left;"><a href="#bookmark1" class="s7">Code</a></p><p style="text-indent: 0pt;text-align: left;"/><p style="padding-top: 5pt;padding-left: 52pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/issues" class="s8" target="_blank">Issues </a><a href="https://github.com/chrishah/MITObim/pulls" class="s10" target="_blank">14 </a><a href="https://github.com/chrishah/MITObim/pulls" class="s8" target="_blank">Pull requests </a><a href="https://github.com/chrishah/MITObim/projects" class="s10" target="_blank">1 </a><a href="https://github.com/chrishah/MITObim/projects" class="s8" target="_blank">Projects </a><a href="https://github.com/chrishah/MITObim/wiki" class="s10" target="_blank">0 </a><a href="https://github.com/chrishah/MITObim/pulse" class="s8" target="_blank">Wiki Insights</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s9" style="text-indent: 0pt;text-align: left;">Dismiss</p><p style="text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><span><img width="33" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_002.png"/></span></p><p class="s12" style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Dismiss</p><p style="text-indent: 0pt;text-align: left;"/><p class="s13" style="padding-left: 165pt;text-indent: 0pt;text-align: center;">Join GitHub today</p><p class="s14" style="padding-top: 1pt;padding-left: 165pt;text-indent: 0pt;line-height: 114%;text-align: center;">GitHub is home to over 36 million developers working together to host and review code, manage projects, and build software together.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="55" height="27" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_003.png"/></span></p><p style="padding-top: 9pt;padding-left: 165pt;text-indent: 0pt;text-align: center;"><a href="https://github.com/join?source=prompt-code" class="s15">Sign up</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">MITObim - mitochondrial baiting and iterative mapping</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s16" style="text-indent: 0pt;text-align: left;">Perl Python Dockerfile</p><p style="text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><span><img width="764" height="50" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_004.png"/></span></p><p style="padding-top: 5pt;padding-left: 43pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commits/master" class="s7" target="_blank">98 </a><a href="https://github.com/chrishah/MITObim/branches" class="s8" target="_blank">commits </a><a href="https://github.com/chrishah/MITObim/branches" class="s7" target="_blank">2 </a><a href="https://github.com/chrishah/MITObim/releases" class="s8" target="_blank">branches </a><a href="https://github.com/chrishah/MITObim/releases" class="s7" target="_blank">2 </a><a href="https://github.com/chrishah/MITObim/graphs/contributors" class="s8" target="_blank">releases </a><a href="https://github.com/chrishah/MITObim/graphs/contributors" class="s7" target="_blank">2 </a><a href="https://github.com/chrishah/MITObim/blob/master/LICENSE" class="s8" target="_blank">contributors MIT</a></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="85" height="22" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_005.png"/></span></p><p class="s6" style="padding-top: 5pt;padding-left: 11pt;text-indent: 0pt;text-align: left;">Branch: master</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s6" style="text-indent: 0pt;text-align: left;">New pull request</p><p style="text-indent: 0pt;text-align: left;"/><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="22" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_006.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="764" height="86" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_007.png"/></span></p><p class="s17" style="padding-top: 2pt;padding-left: 9pt;text-indent: 0pt;text-align: left;">New pull request</p><p style="text-indent: 0pt;text-align: left;"/><p class="s6" style="padding-left: 16pt;text-indent: -10pt;line-height: 159%;text-align: left;"><span><img width="83" height="21" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_008.png"/></span><a href="https://github.com/chrishah/MITObim/find/master" style=" color: black; font-family:&quot;Times New Roman&quot;, serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 10pt;" target="_blank">     </a><a href="https://github.com/chrishah/MITObim/find/master" class="s5" target="_blank">Find </a>File <span style=" color: #FFF;">Clone or download</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 5pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/1a15c5aa1b2cb2b725792cd66dade47df761355b" class="s5" target="_blank">chrishah </a><a href="https://github.com/chrishah/MITObim/commit/1a15c5aa1b2cb2b725792cd66dade47df761355b" class="s20" target="_blank">update zenodo batch</a></p><p class="s21" style="padding-top: 1pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/1a15c5aa1b2cb2b725792cd66dade47df761355b" style=" color: #575F69; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7.5pt;" target="_blank">Latest commit </a><span style=" color: #444D55; font-family:Consolas, monospace; font-style: normal; font-weight: normal; text-decoration: none; font-size: 7pt;">1a15c5a </span>on Sep 4, 2018on Sep 4, 2018</p><p style="text-indent: 0pt;text-align: left;"><br/></p><table style="border-collapse:collapse;margin-left:5.73pt" cellspacing="0"><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/tree/master/devel" class="s23">devel</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/966ebb89ba3c38d5e923d04a012874392781a7e7" class="s24">change result filename</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: right;">9 months ago9 months ago</p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/tree/master/docker" class="s23">docker</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/580daeb051b0a0d2b0cfc04e96690a0718b99bb3" class="s24">change Docker context - installation of GNU parallel</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: right;">9 months ago9 months ago</p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/tree/master/examples" class="s23">examples</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/caf55967517912f6a99e371c30bef1f176946b1b" class="s24">clear notebook outputs</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: right;">3 years ago3 years ago</p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/tree/master/misc_scripts" class="s23">misc_scripts</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/6f92f79bb0861ee66955f809c67506b6ef81a47c" class="s24">add roll functionality to circules</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: right;">2 years ago2 years ago</p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/tree/master/old" class="s23">old</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/17db223b867ef3c4552f5c5ba4407284e74fc088" class="s24">cleanup</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: right;">2 years ago2 years ago</p></td></tr><tr style="height:19pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/blob/master/.gitignore" class="s23">.gitignore</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/27454d98cbbf0e478a8bc42a7e7a39a9ffda8c1f" class="s24">some cosmetics - improve error messages on startup</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-bottom-style:solid;border-bottom-width:1pt;border-bottom-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;text-align: right;">2 years ago2 years ago</p></td></tr><tr style="height:15pt"><td style="width:80pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-left-style:solid;border-left-width:1pt;border-left-color:#DFE1E4"><p style="padding-top: 2pt;padding-left: 11pt;text-indent: 0pt;line-height: 11pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/blob/master/LICENSE" class="s23">LICENSE</a></p></td><td style="width:300pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE"><p style="padding-top: 2pt;padding-left: 28pt;text-indent: 0pt;line-height: 11pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/416a5a5c8b7a10733de0b13371de3649ef20fa35" class="s24">MIT license</a></p></td><td style="width:192pt;border-top-style:solid;border-top-width:1pt;border-top-color:#EAEBEE;border-right-style:solid;border-right-width:1pt;border-right-color:#DFE1E4"><p class="s25" style="padding-top: 2pt;padding-right: 5pt;text-indent: 0pt;line-height: 11pt;text-align: right;">7 years ago7 years ago</p></td></tr></table><p style="text-indent: 0pt;text-align: left;"><span><img width="764" height="882" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_009.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="764" height="83" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_010.png"/></span></p><p class="s9" style="padding-top: 5pt;padding-left: 17pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/966ebb89ba3c38d5e923d04a012874392781a7e7" class="s26" target="_blank">MITObim.pl </a>change result filename <span style=" color: #6A737C;">9 months ago9 months ago</span></p><p class="s9" style="padding-top: 8pt;padding-left: 17pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/1a15c5aa1b2cb2b725792cd66dade47df761355b" class="s26" target="_blank">README.md </a>update zenodo batch <span style=" color: #6A737C;">9 months ago9 months ago</span></p><p class="s9" style="padding-top: 8pt;padding-left: 17pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/chrishah/MITObim/commit/4509b299279fd01ad767fc130b2840727ce956e0" class="s26" target="_blank">testdata1.tgz </a>updated <span style=" color: #6A737C;">6 years ago6 years ago</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s14" style="padding-left: 10pt;text-indent: 0pt;text-align: left;">README.md</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s29" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">MITObim - mitochondrial baiting and iterative mapping</p><p class="s30" style="padding-top: 18pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">VERSIONS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">1.9.1 (stable - relies on MIRA 4.0.2) This version is archived on Zenodo:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">1.6 (stable - relies on MIRA 3.4.1.1)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 195%;text-align: left;">We recommend to use the latest version but see note below regarding availability of the proofreading algorithm. Copyright Christoph Hahn 2012-2018</p><p class="s30" style="padding-top: 2pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">CONTACT</p><p class="s33" style="padding-top: 15pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="https://groups.google.com/forum/%23!forum/mitobim-users" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">We encourage users to post any questions/comments/problems you might have concerning MITObim to our </a>Google Group <a href="https://github.com/chrishah/MITObim/issues" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">or the </a>Github issues<span style=" color: #23292D;">.</span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">or (<i>in emergency situations</i>)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s33" style="padding-left: 34pt;text-indent: 0pt;text-align: left;"><a href="mailto:christoph.hahn@uni-graz.at" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">contact me directly under </a>christoph.hahn@uni-graz.at<span style=" color: #23292D;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">I&#39;ll try respond to you asap! If you don&#39;t hear from me within a few days, please just send another reminder!</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">INTRODUCTION</p><p class="s33" style="padding-top: 15pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="http://nar.oxfordjournals.org/content/41/13/e129" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">This document contains instructions on how to use the MITObim pipeline described in Hahn et al. 2013. The full article can be found </a>here<span style=" color: #23292D;">. Kindly cite the article if you are using MITObim in your work. The pipeline was originally developed for Illumina data, but thanks to the versatility of the MIRA assembler, MITObim supports in principle also data from the Iontorrent, 454 and PacBio sequencing platforms.</span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: justify;">Below you can find a few basic tutorials for how to run MITObim and I encorage you to give them a try with the testdata that comes with this Repo, just to make sure everything is running smoothly on your system. It&#39;ll only take a few minutes and will potentially safe you a lot of time down the line.</p><p class="s33" style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="https://github.com/chrishah/MITObim/tree/master/examples" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">I provide further examples </a>here <span style=" color: #23292D;">as Jupyter notebooks. Get in touch if you feel like sharing your particular MITObim solution and I&#39;d be happy to put it up here, too!</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">PREREQUISITES</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-top: 5pt;padding-left: 52pt;text-indent: 0pt;line-height: 137%;text-align: left;">GNU utilities Perl</p><p style="padding-left: 52pt;text-indent: 0pt;line-height: 12pt;text-align: left;">A running version of MIRA</p><p class="s33" style="padding-top: 2pt;padding-left: 71pt;text-indent: 0pt;line-height: 137%;text-align: left;"><a href="http://sourceforge.net/projects/mira-assembler/files/MIRA/stable/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">MIRA 4.0.2 (for the use with MITObim 1.8 (and newer) - download </a>here<a href="http://sourceforge.net/projects/mira-assembler/files/MIRA/Older%20releases/V3.4.0/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">). MIRA 3.4.1.1 (for the use with MITObim 1.6 - download </a>here<span style=" color: #23292D;">).</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_011.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_012.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_013.png"/></span></p><p class="s33" style="padding-top: 5pt;padding-left: 71pt;text-indent: 0pt;text-align: left;"><a href="http://mira-assembler.sourceforge.net/docs/DefinitiveGuideToMIRA.html" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">Precompiled binaries for MIRA are available for Linux and OSX. An excellent guide to MIRA is available </a>here<span style=" color: #23292D;">.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s33" style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="https://www.docker.com/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">As an alternative, I have generated a </a>docker <a href="https://registry.hub.docker.com/u/chrishah/mitobim/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">image for MITObim (see </a>here<a href="https://www.docker.com/whatisdocker/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">). Find out what docker is </a>here<a href="https://docs.docker.com/installation/#installation" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">. The MITObim image contains a stripped down version of Ubuntu 16.04 and all necessary executables and dependencies to run the latest version of MITObim. Docker is compatible with all major operating systems, including Mac OSX and Windows (see </a>here<span style=" color: #23292D;">). The MITObim image has been tested on Ubuntu, but it should work without problem in any other system where docker was successfully installed.</span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">To install docker on ubuntu should be as easy as:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="41" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_014.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">sudo apt-get install docker.io</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="56" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_015.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="68" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_016.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">You can then specify a working directory on your machine that will be synced with the <span class="s32">/home/data </span>directory in the image and enter the self contained shell environment to run MITObim:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">WORKING_DIR=/your/desired/working/dir</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">sudo docker run -i -t -v $WORKING_DIR/:/home/data chrishah/mitobim /bin/bash</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s33" style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="https://registry.hub.docker.com/u/chrishah/mitobim/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">NOTE that the first startup may take a couple of minutes because the above command will first download the image from </a><a href="https://registry.hub.docker.com/u/chrishah/mitobim/" class="a" target="_blank">docker </a>hub<span style=" color: #23292D;">. Subsequent launches will take only seconds.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">COMMENT</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="688" height="1" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_017.png"/></span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: justify;">The proofreading option described in the paper is at the moment disabled in MITObim 1.8 and will be enabled once I got a chance to thoroughly test its behavior with MIRA 4. If you are planning to use the proofreading functunality please refer to MITObim 1.6 for the time being. Sorry for the inconvenience!</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">General introduction to MITObim</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="688" height="1" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_018.png"/></span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">The MITObim procedure (mitochondrial baiting and iterative mapping) represents a highly efficient approach to assembling novel mitochondrial genomes of non-model organisms directly from total genomic DNA derived NGS reads. Labor intensive long-range PCR steps prior to sequencing are no longer required. MITObim is capable of reconstructing mitochondrial genomes without the need of a reference genome of the targeted species by relying solely on (a) mitochondrial genome information of more distantly related taxa or (b) short mitochondrial barcoding sequences (seeds), such as the commonly used cytochrome-oxidase subunit 1 (COI), as a starting reference.</p><p class="s33" style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="http://nar.oxfordjournals.org/content/41/13/e129" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">The script is performing three steps and iteratively repeating them: (i) Deriving reference sequence from previous mapping assembly, (ii) in silico baiting using the newly derived reference (iii) previously fished reads are mapped to the newly derived reference leading to an extension of the reference sequence. For more details please refer to </a>Hahn et al. 2013<span style=" color: #23292D;">. Detailed examples are demonstrated in the TUTORIALS section below.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">TUTORIALS</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="688" height="1" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_019.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="92" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_020.png"/></span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">The following tutorials are designed for users with little Unix and no previous MIRA experience. Tutorials I &amp; II will demonstrate how to recover the complete mitochondrial genome of <i>Thymallus thymallus </i>using the mitochondrial genome of <i>Salvelinus alpinus </i>as a starting reference. Tutorial III achieves the same goal using solely a ~700 bp barcoding sequence as initial seed reference. For a quick exploration of your own data I recommend trying something along the lines of Tutorial II. Tutorial IV (to be added soon) uses a proofreading procedure to specifically reconstruct two mitochondrial genomes from a mixed sample containing genomic reads from two species. For convenience I always refer to MITObim.pl in the tutorials - The individual user will have to call the respective version of MITObim (e.g. <span class="s32">MITObim_1.6.pl </span>) during trying the tutorials. MITObim 1.7 (and newer) might finish some of the tutorials with slighly less iterations than described in the tutorials. Dont worry!</p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_021.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_022.png"/></span></p><p style="padding-top: 6pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">Preparations:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_023.png"/></span></p><p class="s33" style="padding-left: 52pt;text-indent: 0pt;line-height: 117%;text-align: left;"><a href="http://sourceforge.net/projects/mira-assembler/files/MIRA/stable/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">download MIRA (MIRA 4 for the use with MITObim 1.7 (and newer) from </a>here <a href="http://sourceforge.net/projects/mira-assembler/files/MIRA/Older%20releases/V3.4.0/" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">or MIRA 3.4.1.1 for the use with MITObim 1.6 from </a>here<a href="http://mira-assembler.sourceforge.net/docs/DefinitiveGuideToMIRA.html" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">. Precompiled binaries are available for Linux and OSX. Help can be found </a>here<span style=" color: #23292D;">. You ll need to put the directory containing the MIRA executables in your PATH in order to successfully use MITObim.pl. If you can&#39;t or won&#39;t do that you can also tell MITObim where to find the correct MIRA binaries via the --mirapath option.</span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_024.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="130" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_025.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="226" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_026.png"/></span></p><p style="padding-top: 9pt;padding-left: 52pt;text-indent: 0pt;line-height: 117%;text-align: left;">download the MITObim wrapper script and the testdata from Github, e.g. download the entire MITObim repository as zip archive (use the button on the Github page) or use git on the command line ( <span class="s32">git clone --recursive git://github.com/chrishah/MITObim.git </span>).</p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_027.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="128" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_028.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_029.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="82" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_030.png"/></span></p><p style="padding-top: 9pt;padding-left: 52pt;text-indent: 0pt;line-height: 117%;text-align: left;">make the MITObim.pl executable ( <span class="s32">chmod a+x MITObim.pl </span>) and extract the contents of the testdata archives ( <span class="s32">tar xvfz testdata1.tgz </span>)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_031.png"/></span></p><p style="padding-top: 9pt;padding-left: 52pt;text-indent: 0pt;text-align: left;">OR just fire up the self-contained docker image (see prerequisites above).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Test the wrapper script by doing:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="41" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_032.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ ~/PATH/TO/MITObim.pl</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="591" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_033.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_034.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="49" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_035.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">which should display the usage (NOTE: From MITObim 1.7 onwards the <span class="s32">-strain </span>flag has been renamed to <span class="s32">-sample </span>), e.g.:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">MITObim - mitochondrial baiting and iterative mapping version 1.9.1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">usage: ./MITObim.pl &lt;parameters&gt;</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-top: 3pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">parameters:</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">-start &lt;int&gt; iteration to start with (default=0, when using &#39;-quick&#39; reference)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">-end &lt;int&gt; iteration to end with (default=startiteration, i.e. if not specified</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">otherwise stop after 1 iteration)</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">-sample &lt;string&gt; sampleID (please don&#39;t use &#39;.&#39; in the sampleID). If resuming, the sampleID needs to be identical to that of the previous iteration / MIRA assembly.</p><p class="s32" style="padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">-ref &lt;string&gt; referenceID. If resuming, use the same as in previous iteration/initial MIRA assembly.</p><p class="s32" style="padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">-readpool &lt;FILE&gt; readpool in fastq format (*.gz is also allowed). read pairs need to be interleaved for full functionality of the &#39;-pair&#39; option below.</p><p class="s32" style="padding-left: 115pt;text-indent: 0pt;line-height: 9pt;text-align: left;">-quick &lt;FILE&gt; reference sequence to be used as bait in fasta format</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">-maf &lt;FILE&gt; extracts reference from maf file created by previous MITObim iteration/MIRA assembly (resume)</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-top: 3pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">optional:</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--kbait &lt;int&gt; set kmer for baiting stringency (default: 31)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--platform specify sequencing platform (default: &#39;solexa&#39;; other options:</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">&#39;iontor&#39;, &#39;454&#39;, &#39;pacbio&#39;)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--denovo runs MIRA in denovo mode</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">--pair extend readpool to contain full read pairs, even if only one member was baited (relies on /1 and /2 header convention for read pairs) (default: no).</p><p class="s32" style="padding-left: 115pt;text-indent: 0pt;line-height: 9pt;text-align: left;">--verbose show detailed output of MIRA modules (default: no)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--split split reference at positions with more than 5N (default: no)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--help shows this helpful information</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--clean retain only the last 2 iteration directories (default: no)</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">--trimreads trim data (default: no; we recommend to trim beforehand and feed MITObim with pre trimmed data)</p><p class="s32" style="padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">--trimoverhang trim overhang up- and downstream of reference, i.e. don&#39;t extend the bait, just re-assemble (default: no)</p><p class="s32" style="padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">--mismatch &lt;int&gt; number of allowed mismatches in mapping - only for illumina data (default: 15% of avg. read length)</p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_036.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_037.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="246" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_038.png"/></span></p><p class="s32" style="padding-top: 7pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--min_cov &lt;int&gt; minimum average coverage of contigs to be retained (default: 0 - off)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--min_len &lt;int&gt; minimum length of contig to be retained as backbone (default: 0 - off)</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">--mirapath &lt;string&gt; full path to MIRA binaries (only needed if MIRA is not in PATH)</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">--redirect_tmp redirect temporary output to this location (useful in case you are running MITObim on an NFS mount)</p><p class="s32" style="padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">--NFS_warn_only allow MIRA to run on NFS mount without aborting - warn only (expert option - see MIRA documentation &#39;check_nfs&#39;)</p><p class="s32" style="padding-left: 115pt;text-indent: 0pt;line-height: 9pt;text-align: left;">--version display MITObim version</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-top: 3pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">examples:</p><p class="s32" style="padding-top: 2pt;padding-left: 115pt;text-indent: 0pt;text-align: left;">./MITObim.pl -start 1 -end 5 -sample StrainX -ref reference-mt -readpool</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">illumina_readpool.fastq -maf initial_assembly.maf</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 72pt;line-height: 125%;text-align: left;">./MITObim.pl -end 10 -quick reference.fasta -sample StrainY -ref reference-mt -readpool illumina_readpool.fastq</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">The archive testdata1 contains three files:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="260" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_039.png"/></span></p><ol id="l1"><li style="padding-left: 52pt;text-indent: -9pt;line-height: 117%;text-align: left;"><p style="display: inline;"><span class="s34">	</span><span class="s32">Tthymallus-150bp-300sd50-interleaved.fastq </span>- 6000 simulated illumina reads (read length 150 bp, insert size 300 +- 50 bp) for the mitochondrial genome of <i>T. thymallus </i><a href="http://nar.oxfordjournals.org/content/41/13/e129" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">as discussed in </a><a href="http://nar.oxfordjournals.org/content/41/13/e129" class="a" target="_blank">Hahn et </a><span style=" color: #0366D5;">al</span>.</p><p style="text-indent: 0pt;text-align: left;"><span><img width="212" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_040.png"/></span></p></li><li style="padding-top: 9pt;padding-left: 52pt;text-indent: -11pt;line-height: 117%;text-align: left;"><p style="display: inline;"><span class="s34">	</span><span class="s32">Salpinus-mt-genome-NC_000861.fasta </span>- mitochondrial genome of <i>S. alpinus </i>in fasta format downloaded from Genbank (accession NC000861).</p><p style="text-indent: 0pt;text-align: left;"><span><img width="230" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_041.png"/></span></p></li><li style="padding-top: 9pt;padding-left: 55pt;text-indent: -14pt;text-align: left;"><p style="display: inline;"><span class="s32">Tthymallus-COI-partial-HQ961018.fasta </span>- partial COI sequence of <i>T. thymallus </i>(Genbank acc. HQ961018).</p></li></ol><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;text-align: left;">TUTORIAL I: reconstruction of a mitochondrial genome using a two step procedure</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="688" height="1" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_042.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_043.png"/></span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">One strategy for successful MITObim performance is to prepare an initial reference from the conserved regions of a distantly related mitochondrial reference genome using a MIRA mapping assembly in a first step. In a second step the MITObim wrapper script is applied to this newly constructed reference in order to reconstruct the entire mitochondrial genome. Impatient users are encouraged to move directly to Tutorials II &amp; III below which show examples for using the <span class="s32">--quick </span>strategy - a single step procedure - i.e. they don&#39;t require the initial mapping assembly to be performed manually.</p><ol id="l2"><li style="padding-top: 9pt;padding-left: 43pt;text-indent: -9pt;text-align: left;"><p style="display: inline;">Initial mapping assembly using MIRA:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Create a directory to work in and change into this directory:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="56" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_044.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ mkdir tutorial1</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ cd tutorial1</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">mapping assembly with MIRA 3.4.1.1:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="87" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_045.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="100" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_046.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="142" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_047.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">MIRA 3.4.1.1 expects certain files to be present in your working directory, which are named according to the name of your MIRA project and certain conventions. Let&#39;s say the project name of your choice for the initial mapping assembly is <span class="s32">initial-mapping- testpool-to-Salpinus-mt </span>. One could either simply copy and rename the files accordingly in your working directory or create symbolic links in the current working directory pointing to the actual files (replace &quot;cp&quot; with &quot;ln -s&quot; in the following command).</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ cp /PATH/TO/testdata1/Tthymallus-150bp-300sd50-interleaved.fastq initial-mapping-testpool-to- Salpinus-mt_in.solexa.fastq</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ cp /PATH/TO/testdata1/Salpinus-mt-genome-NC_000861.fasta initial-mapping-testpool-to-Salpinus- mt_backbone_in.fasta</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">run MIRA 3.4.1.1:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;"><span><img width="680" height="19" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_048.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_049.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_050.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="43" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_051.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ mira --project=initial-mapping-testpool-to-Salpinus-mt --job=mapping,genome,accurate,solexa -</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">MI:somrnl=0 -AS:nop=1 -SB:bft=fasta:bbq=30:bsn=Salpinus-mt-genome SOLEXA_SETTINGS -CO:msr=no -SB:dsn=testpool</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">or</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">mapping assembly with MIRA 4:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="56" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_052.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="32" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_053.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="50" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_054.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="32" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_055.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">MIRA 4 does not depend on the naming conventions of MIRA 3.4 any more. MIRA 4 uses a so called manifest file in which you can specifiy the paths to your data (see below). Let us for the sake of this example create symbolic links, nevertheless. <i>NOTE </i>that we change the extension of the reference fasta file to <span class="s32">*.fa </span>. The reason is that MIRA will per default expect quality data for files with the extension <span class="s32">*.fasta </span>. The file extension <span class="s32">*.fa </span>tells MIRA to continue even without quality data.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ ln -s /PATH/TO/testdata1/Tthymallus-150bp-300sd50-interleaved.fastq reads.fastq</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ ln -s /PATH/TO/testdata1/Salpinus-mt-genome-NC_000861.fasta reference.fa</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="88" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_056.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="86" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_057.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">create a manifest file (named e.g. <span class="s32">manifest.conf </span><a href="http://mira-assembler.sourceforge.net/docs/DefinitiveGuideToMIRA.html" style=" color: #23292D; font-family:&quot;Segoe UI Light&quot;, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 9pt;" target="_blank">) specifying the parameters for your MIRA assembly (see section 3.4 </a><span style=" color: #0366D5;">here </span>for details) in your favourite text editor or just type (or copy and paste) the following command:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ echo -e &quot;\n#manifest file for basic mapping assembly with illumina data using MIRA 4\n\nproject = initial-mapping-testpool-to-Salpinus-mt\n\njob=genome,mapping,accurate\n\nparameters = -NW:mrnl=0 -AS:nop=1 SOLEXA_SETTINGS -CO:msr=no\n\nreadgroup\nis_reference\ndata = reference.fa\nstrain = Salpinus-mt- genome\n\nreadgroup = reads\ndata = reads.fastq\ntechnology = solexa\nstrain = testpool\n&quot; &gt; manifest.conf</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Whatever you do, the manifest.conf file should eventually look more or less as follows:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="321" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_058.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ head -n 20 manifest.conf</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 250%;text-align: left;">#manifest file for basic mapping assembly with illumina data using MIRA 4 project = initial-mapping-testpool-to-Salpinus-mt job=genome,mapping,accurate</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 9pt;text-align: left;">parameters = -NW:mrnl=0 -AS:nop=1 SOLEXA_SETTINGS -CO:msr=no</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">readgroup is_reference</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 9pt;text-align: left;">data = reference.fa</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">strain = Salpinus-mt-genome</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-top: 4pt;padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">readgroup = reads data = reads.fastq technology = solexa strain = testpool</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">run MIRA 4:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="41" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_059.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ mira manifest.conf</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Look what happened:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="70" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_060.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ ls -hlrt total 2.8M</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 9pt;text-align: left;">-rw-r--r-- 1 chrishah users 17K Oct 21 22:50 initial-mapping-testpool-to-Salpinus-mt_backbone_in.fasta</p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_061.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_062.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="43" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_063.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-rw-r--r-- 1 chrishah users 2.7M Oct 21 22:51 initial-mapping-testpool-to-Salpinus-mt_in.solexa.fastq</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">drwxr-xr-x 6 chrishah users 4 Oct 21 22:51 initial-mapping-testpool-to-Salpinus-mt_assembly</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="119" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_064.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="296" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_065.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">The successfull MIRA instance created a directory <span class="s32">initial-mapping-testpool-to-Salpinus-mt_assembly </span>, containing four directories:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ ls -hlrt initial-mapping-testpool-to-Salpinus-mt_assembly/ total 2.0K</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 9pt;text-align: left;">drwxr-xr-x 2 chrishah users 2 Oct 21 22:51 initial-mapping-testpool-to-Salpinus-mt_d_chkpt</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">drwxr-xr-x 2 chrishah users 15 Oct 21 22:51 initial-mapping-testpool-to-Salpinus-mt_d_tmp</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">drwxr-xr-x 2 chrishah users 9 Oct 21 22:51 initial-mapping-testpool-to-Salpinus-mt_d_results</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">drwxr-xr-x 2 chrishah users 8 Oct 21 22:51 initial-mapping-testpool-to-Salpinus-mt_d_info</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="290" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_066.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="52" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_067.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="250" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_068.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">The newly constructed reference is contained in the file <span class="s32">initial-mapping-testpool-to-Salpinus-mt_out.maf </span>in the <span class="s32">initial- mapping-testpool-to-Salpinus-mt_d_results </span>directory.</p></li><li style="padding-top: 9pt;padding-left: 43pt;text-indent: -9pt;text-align: left;"><p style="display: inline;">Baiting and iterative mapping using the MITObim.pl script:</p></li></ol><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="72" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_069.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 195%;text-align: left;">Run the wrapper script as follows (standard output will be written into the file log - <i>approximate runtime: 2 min</i>): for MITObim 1.6:</p><p class="s32" style="padding-top: 10pt;padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ /PATH/TO/MITObim.pl -start 1 -end 10 -strain testpool -ref Salpinus_mt_genome -readpool initial- mapping-testpool-to-Salpinus-mt_in.solexa.fastq -maf initial-mapping-testpool-to-Salpinus-mt_assembly/initial- mapping-testpool-to-Salpinus-mt_d_results/initial-mapping-testpool-to-Salpinus-mt_out.maf &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.7 and later:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="72" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_070.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ /PATH/TO/MITObim.pl -start 1 -end 10 -sample testpool -ref Salpinus_mt_genome -readpool reads.fastq</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-maf initial-mapping-testpool-to-Salpinus-mt_assembly/initial-mapping-testpool-to-Salpinus- mt_d_results/initial-mapping-testpool-to-Salpinus-mt_out.maf &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">NOTE:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="5" height="5" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_071.png"/></span></p><p style="padding-left: 52pt;text-indent: 0pt;text-align: left;">The strain/sample name needs to be the same as used in the initial MIRA assembly.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">After the process has finished looking into the log file will yield something like:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="150" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_072.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ tail log</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 23pt;text-align: left;">End of assembly process, thank you for using MIRA. readpool contains 6000 reads</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">contig length: 16667</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">MITObim has reached a stationary read number after 8 iterations!!</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="70" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_073.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">Allthough you have asked for 10 iterations MITObim stopped the process after fewer iterations because it has detected a stationary readnumber. It will have created a number of iteration* directories. Each of which is organized like the working directory above (initial mapping assembly), containing a MIRA assembly directory plus the referernce and readpool used in the respective iteration:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ ls -hlrt iteration1/ total 2.3M</p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 9pt;text-align: left;">-rw-r--r-- 1 chrishah users 17K Oct 21 23:45 testpool-Salpinus_mt_genome_backbone_in.fasta</p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_074.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="1" height="974" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_075.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="59" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_076.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-rw-r--r-- 1 chrishah users 383K Oct 21 23:45 hashstat.bin</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">-rw-r--r-- 1 chrishah users 1.9M Oct 21 23:45 testpool-Salpinus_mt_genome_in.solexa.fastq</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">drwxr-xr-x 6 chrishah users 4 Oct 21 23:45 testpool-Salpinus_mt_genome_assembly</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="56" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_077.png"/></span></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">A fasta file containing the complete mitochondondrial genome of <i>T. thymallus </i>can be found in the final iteration directory. See its contents by doing e.g.:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ less iteration8/testpool-Salpinus_mt_genome_assembly/testpool- Salpinus_mt_genome_d_results/testpool-Salpinus_mt_genome_out.unpadded.fasta</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">Congratulations!!!</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s31" style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">IMPORTANT NOTE<span class="p">: For real data this strategy might require substantial computational resources depending on the size of the illumina readpool (Memory requirements can be estimated using the miramem program shipping with MIRA). This increased memory consumption can be bypassed by limitating the readpool e.g. via an initial in-silico baiting step using mirabait (in-silico baiting program shipping with MIRA). While this strategy will impair the initial sensitivity of the approach and might therefore require more iterations to finish, it works reasonably well if a not too distantly related reference is available. This strategy can be performed by using the --quick flag, together with providing a reference sequence in fasta format. A detailed example can be found in TUTORIAL II below.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s30" style="padding-left: 34pt;text-indent: 0pt;line-height: 94%;text-align: left;">TUTORIAL II - direct reconstruction without prior mapping assembly using the --quick option</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="688" height="1" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_078.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="56" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_079.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><span><img width="92" height="16" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_080.png"/></span></p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">This TUTORIAL illustrates the quick strategy that I usually use in a first test. It bypasses the intial mapping assembly required in TUTORIAL I, i.e. you only need a reference in fasta format and your reads. To finish a mitochondrial genome it usually takes more iterations than TUTORIAL I above because the initial mapping assembly is less thorough. The big advantage is that the whole process can be run on a standard Desktop computer due to the substantial reduction in the number of reads to be dealt with already in the first iteration. Run the MITObim.pl script with the <span class="s32">--quick option </span>, providing a reference in fasta format (<i>approximate runtime: 4 min</i>):</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ mkdir tutorial2</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ cd tutorial2</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.6:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="72" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_081.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ /PATH/TO/MITObim.pl -start 1 -end 30 -strain testpool -ref Salpinus_mt_genome -readpool</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">~/PATH/TO/testdata1/Tthymallus-150bp-300sd50-interleaved.fastq --quick ~/PATH/TO/testdata1/Salpinus-mt-genome-</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">NC_000861.fasta &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.7 and later:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="text-indent: 0pt;text-align: left;"><span><img width="688" height="72" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_082.png"/></span></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ /PATH/TO/MITObim.pl -start 1 -end 30 -sample testpool -ref Salpinus_mt_genome -readpool</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">~/PATH/TO/testdata1/Tthymallus-150bp-300sd50-interleaved.fastq --quick ~/PATH/TO/testdata1/Salpinus-mt-genome- NC_000861.fasta &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">You will find that with this approach MITObim will reconstruct the mitochondrial genome and reach a stationary number of mitochondrial reads only after 15 (or less) iterations. The result should nevertheless be equal to that obtained in TUTORIAL I and we were able to bypass the increased memory requirements of the initial MIRA mapping assembly.</p><p style="padding-top: 9pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">CONGRATULATIONS!! ..but, there is more..</p><p style="text-indent: 0pt;text-align: left;"><span><img width="764" height="860" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_083.png"/></span></p><p class="s30" style="padding-top: 4pt;padding-left: 34pt;text-indent: 0pt;text-align: left;">TUTORIAL III - reconstructing mt genomes from mt barcode seeds</p><p style="padding-top: 15pt;padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">This tutorial reconstructs the mt genome of <i>T. thymallus </i>solely using a partial mitochondrial COI sequence as starting seed. <i>NOTE </i>that we also use the new <span class="s32">--clean </span>option which tells MITObim to always only keep the latest two iteration directories to save space (<i>approximate runtime: 20 min</i>):</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-1.4$ mkdir tutorial3</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-1.4$ cd tutorial3</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.6:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-1.4$ ~/PATH/TO/MITObim.pl -strain testpool -ref Tthymallus-COI -readpool ~/PATH/TO/testdata1/Tthymallus- 150bp-300sd50-interleaved.fastq --quick ~/PATH/TO/testdata1/Tthymallus-COI-partial-HQ961018.fasta -end 100 -- noshow --clean &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.7 and later:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-1.4$ ~/PATH/TO/MITObim.pl -sample testpool -ref Tthymallus-COI -readpool ~/PATH/TO/testdata1/Tthymallus- 150bp-300sd50-interleaved.fastq --quick ~/PATH/TO/testdata1/Tthymallus-COI-partial-HQ961018.fasta -end 100 -- clean &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">MITObim reconstructs the mitchondrial genome in 82 (or less) iterations.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;line-height: 117%;text-align: left;">For &quot;well behaved&quot; datasets the standard mapping assembly can be substituted by a <i>de novo </i>assembly ( <span class="s32">--denovo </span>flag). Utilizing read pair information ( <span class="s32">--paired </span>flag) can further speed up the reconstruction if run in <i>de novo </i>mode. This however will not decrease the number of necessary iterations in standard mapping mode. Test this strategy, like so (<i>approximate runtime: 10 min</i>):</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ mkdir tutorial3-denovo</p><p class="s32" style="padding-top: 2pt;padding-left: 43pt;text-indent: 0pt;text-align: left;">-bash-4.1$ cd tutorial3-denovo</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.6:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ ~/PATH/TO/MITObim.pl -strain testpool -ref Tthymallus-COI -readpool ~/PATH/TO/testdata1/Tthymallus- 150bp-300sd50-interleaved.fastq --quick ~/PATH/TO/testdata1/Tthymallus-COI-partial-HQ961018.fasta -end 50 -- noshow --denovo --paired --clean &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">for MITObim 1.7 and later:</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s32" style="padding-left: 43pt;text-indent: 0pt;line-height: 125%;text-align: left;">-bash-4.1$ ~/PATH/TO/MITObim.pl -sample testpool -ref Tthymallus-COI -readpool ~/PATH/TO/testdata1/Tthymallus- 150bp-300sd50-interleaved.fastq --quick ~/PATH/TO/testdata1/Tthymallus-COI-partial-HQ961018.fasta -end 50 -- denovo --paired --clean &amp;&gt; log</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 34pt;text-indent: 0pt;text-align: left;">This strategy reconstructs the correct mitochondrial genome in only 31 (or less) iterations.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 1pt;text-align: left;"><span><img width="764" height="1" alt="image" src="GitHub - chrishahMITObim MITObim - mitochondrial baiting and iterative mapping/Image_084.png"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s35" style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">© 2019 GitHub, Inc.</p><p style="padding-top: 1pt;padding-left: 5pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/site/terms" class="s36">Terms</a></p><p style="padding-top: 1pt;padding-left: 5pt;text-indent: 0pt;text-align: left;"><a href="https://github.com/site/privacy" class="s36">Privacy</a></p><p style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="https://githubstatus.com/" class="s36" target="_blank">Security </a><a href="https://help.github.com/" class="s36" target="_blank">Status Help</a></p><p style="padding-left: 5pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="https://github.com/pricing" class="s36" target="_blank">Contact GitHub Pricing</a></p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"><a href="https://developer.github.com/" class="s36">API</a></p><p style="padding-top: 1pt;padding-left: 5pt;text-indent: 0pt;line-height: 112%;text-align: left;"><a href="https://github.blog/" class="s36" target="_blank">Training </a><a href="https://github.com/about" class="s36" target="_blank">Blog About</a></p></body></html>
